installer/windows/Product.wxs

```diff
@@ -79,7 +79,12 @@
     </ComponentGroup>
     
     <Property Id="WIXUI_INSTALLDIR">INSTALLFOLDER</Property>
-    <Property Id="STORJ_IDENTITYDIR">IDENTITYDIR</Property>
+
+    <Property Id="STORJ_DEFAULTIDENTITYDIR"  >
+       <DirectorySearch Id="search1" Depth="0" Path="[AppDataFolder]" />
+       <DirectorySearch Id="search2" Depth="0" Path="[AppDataFolder]\Storj\Identity\storagenode" />
+      </Property>
+    <Property Id="STORJ_IDENTITYDIR"><![CDATA[STORJ_DEFAULTIDENTITYDIR]]></Property>
```

Comment:
Oh, I didn't know about `DirectorySearch`. This really changes the default value in the field, but it does not modify the `IDENTITYDIR` Directory object. The validation is done on the Directory object, not on the `STORJ_IDENTITYDIR` property, which is used to pass the Directory object to the UI.

Instead of `DirectorySearch`, try with `SetDirectory`, e.g.
```
<SetDirectory Action="SetIdentityDir" Id="IDENTITYDIR" Value="[AppDataFolder]\Storj\Identity\storagenode">NOT Installed</SetDirectory>
```
More info here: https://stackoverflow.com/questions/43714589/how-to-use-wixtoolset-directorysearch